src/ophys_etl/modules/segmentation/detect/feature_vector_utils.py

```diff
@@ -0,0 +1,246 @@
+from typing import Tuple, Optional, List
+import numpy as np
+
+
+def choose_extreme_pixels(
+        image_data: np.ndarray,
+        delta_z: List[float],
+        pixel_ignore: Optional[np.ndarray] = None) -> List[Tuple[int, int]]:
+    """
+    For a specified list of values delta_z, choose the pixels in an
+    image that are that many standard deviations from the maximum
+    and minimum flux values
+
+    i.e. if delta_z = [1, 2]
+
+    The output will be the row, column coordinates of the pixels
+    at
+
+    [flux_min + 2*sigma,
+     flux_min + 1*sigma,
+     flux_max - 2*sigma,
+     flux_max - 1*sigma]
+
+    Note: for each delta_z value, only one pixel will be found
+    using np.argmin(flux-target_flux)
+
+    Parameters
+    ----------
+    image_data: np.ndarray
+        Image data used for determining flux of pixels
+
+    delta_z: List[float]
+        List of differences, in z-score, from extremities
+        to find
+
+    pixel_ignore: Optional[np.ndarray]:
+        A boolean mask marked True at any pixels
+        that should be ignored, presumably because
+        they have already been selected as ROI pixels.
+        (default: None)
+
+    Returns
+    -------
+    interesting_points: List[Tupple[int, int]]
+        Coordinates of interesting points in row, column space.
+        Points will be in ascending flux order.
+
+    """
+    if pixel_ignore is not None:
+        if pixel_ignore.shape != image_data.shape:
+            msg = f"pixel_ignore.shape {pixel_ignore.shape}\n"
+            msg += f"image_data.shape {image_data.shape}\n"
+            msg += "These should be the same"
+            raise RuntimeError(msg)
+
+    image_flat = image_data.flatten()
+    if pixel_ignore is not None:
+        pixel_ignore_flat = pixel_ignore.flatten()
+    else:
+        pixel_ignore_flat = np.zeros(len(image_flat), dtype=bool)
+
+    pixel_indices = np.arange(len(image_flat), dtype=int)
+    valid_pixels = np.logical_not(pixel_ignore_flat)
+    image_flat = image_flat[valid_pixels]
+    pixel_indices = pixel_indices[valid_pixels]
+
+    image_max = image_flat.max()
+    image_min = image_flat.min()
+    t25, t75 = np.quantile(image_flat, (0.25, 0.75))
```

Comment:
I've seen this estimate a few times. It would probably be good to add 
```
# estimate standard deviation by interquartile range
```
so that later, is is clear.

Reply:
I've moved all of the invocations of this formula into a utility function
```
stats_utils.estimate_std_from_interquartile_range
```
Gives us one place to fix everything if we decide this is the wrong move in the future.